src/Symfony/Bundle/FrameworkBundle/FrameworkBundle.php

```diff
@@ -82,7 +82,7 @@ public function build(ContainerBuilder $container)
         $container->addCompilerPass(new FragmentRendererPass(), PassConfig::TYPE_AFTER_REMOVING);
         $container->addCompilerPass(new SerializerPass());
 
-        if ($container->getParameter('kernel.debug')) {
+        if ($container->hasParameter('kernel.debug') && $container->getParameter('kernel.debug')) {
```

Comment:
this looks wrong. The Kernel class ensures that the kernel parameters are always set in the container when calling the bundle classes. You should not remove this assumption IMO.